drake/systems/plants/KinematicsCache.h

```diff
@@ -121,6 +126,44 @@ class KinematicsCache
     }
   }
 
+  template <typename Derived>
+  Eigen::Matrix<typename Derived::Scalar, Derived::RowsAtCompileTime, Eigen::Dynamic> transformVelocityMappingToPositionDotMapping(const Eigen::MatrixBase<Derived>& mat) const
```

Comment:
ah.  i thought this would take in v and output qdot.  better add documentation here, too?  is there a common case where you wouldn't want to simply pass in the identity matrix?